src/zcl/definition/cluster.ts

```diff
@@ -5029,6 +5029,16 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    clipsalWiserFanSwitchConfigurationCluster: {
+        ID: 0xFC04,
+        manufacturerCode: ManufacturerCode.SCHNEIDER,
+        attributes: {
+            switchIndication: {ID: 0x0002, type: DataType.uint8},
```

Comment:
We can probably also drop the `switch` prefix here as it already clear we're talking about a switch from the cluster name.

Reply:
Hindsight, this probably should be named Led indication, rather than switch indication. Will adjust once naming of cluster is agreed upon.